l10n/de_DE.js

```diff
@@ -68,6 +68,8 @@ OC.L10N.register(
     "Changes in the <strong>Deck app</strong>" : "Änderungen in der <strong>Deck-App</strong>",
     "A <strong>comment</strong> was created on a card" : "Ein <strong>Kommentar</strong> zu einer Karte wurde erstellt",
     "Upcoming cards" : "Anstehende Karten",
+    "Upcoming cards today" : "Heute anstehende Karten",
```

Comment:
Translations should be handled through transifex. Changes done manually will be dropped on the next nightly sync with transifex.

Reply:
Okay, where do find the transifex repo?

Reply:
https://www.transifex.com/nextcloud/nextcloud/deck/ :)

Reply:
I've just deleted this lines.